### PR TITLE
Finished problems 13 and last part of 19.

### PR DESCRIPTION
**Pull request by ikhat**
Tying up some loose ends here. 

Problem 13 I had skipped for some reason earlier. Probably because it wants pictures. 

As for Problem 19, I asked a question about the last part earlier, and now I've added a solution for it.

### CODE REVIEW

File: problemsets-mat223/linearalbegra.tex [hunk 1]
```diff
@@ -1303,11 +1303,34 @@ \section*{Span}
...
+				\[
+					\mat{x\\y}
+					=\frac{x}{2}\left(\mat{1\\1}+\mat{1\\-1}\right)+\frac{y}{2}\left(\mat{1\\1}-\mat{1\\-1}\right)
+					=\frac{x+y}{2}\vec v_1 + \frac{x-y}{2}\vec v_2
+				\]
+			\end{solution}
+		\item Describe $\Span\Set{\vec v_1,\vec v_3}$.
+			\begin{solution}[inline]
+				$\Span\Set{\vec v_1,\vec v_3} = \Span\Set{\vec v_1}$, 
+				a line through the origin along $\vec v_1$.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: along -> with direction

File: problemsets-mat223/linearalbegra.tex [hunk 2]
```diff
@@ -1719,6 +1742,10 @@ \section*{Task 1.3: The Magic Carpet, Getting Back Home}
 				vectors. 
 			\end{solution}
 		\item Is $\Set{\vec u,\vec v,\vec w,\vec r}$ linearly independent?
+			\begin{solution}
+				No. We already know $\Set{\vec u,\vec v,\vec w}$ is not linearly
+				independent, so no set that contains it can be linearly independent.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Be explicit from the def. "From before we know u+v-w=0 and therefore u+v-w+0r=0 is a non-trivial linear combination of ..." Students at this point will have the intuition about subsets but wouldn't be able to write down a proof.

File: problemsets-mat223/linearalbegra.tex [hunk 3]
```diff
@@ -1719,6 +1742,11 @@ \section*{Task 1.3: The Magic Carpet, Getting Back Home}
 				vectors. 
 			\end{solution}
 		\item Is $\Set{\vec u,\vec v,\vec w,\vec r}$ linearly independent?
+			\begin{solution}
+				No. We know from earlier that $\vec u+\vec v+2\vec w=\vec 0$, and
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: -2\vec w

Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: It's +, right? The original equation is:

3u + 2v + w = 2u + v - w

Bringing everything to the left side:

(3u - 2u) + (2v - v) + (w - (-w)) = 0

Comment (siefkenj) [reply problemsets-mat223/linearalbegra.tex]: My mistake!

**Pull request merged**